src/app/checkout/navigateToOrderConfirmation.tsx

```diff
@@ -2,16 +2,22 @@ import { noop } from 'lodash';
 
 import { isBuyNowCart } from '../common/utility';
 
-export default function navigateToOrderConfirmation(location = window.location, orderId?: number): Promise<never> {
+export default function navigateToOrderConfirmation(isBuyNowCartEnabled?: boolean, orderId?: number): Promise<never> {
```

Comment:
We removing the `location` param? 
Also `isBuyNowCartEnabled: boolean`

Reply:
I see `location` is only used in testing, so leaving it there looks kind of redundant.
If `isBuyNowCartEnabled` is not optional, [this line](https://github.com/bigcommerce/checkout-js/blob/a7fa190d45b55500eeacb7d7c837f5b771578d85/src/app/customer/customWalletButton/ApplePayButton.tsx#L21) will complain.

Reply:
In that case maybe need to open a PR for SDK to accept that param,https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/core/src/customer/strategies/apple-pay/apple-pay-customer-initialize-options.ts#L27

`    onPaymentAuthorize(isBuyNowCartEnabled: boolean): void;
`